packages/amplify-category-api/package.json

```diff
@@ -30,12 +30,14 @@
   },
   "dependencies": {
     "@aws-amplify/graphql-auth-transformer": "2.1.12",
-    "@aws-amplify/graphql-schema-generator": "0.1.8",
+    "@aws-amplify/graphql-schema-generator": "0.2.0",
     "@aws-amplify/graphql-transformer": "0.0.7",
```

Comment:
This shouldn't be changing, right?

Reply:
Same reason as why I have upgraded the version on the package. This is done to avoid version conflict error encountered while publishing a tagged version.